scripts/cloudbees-demo.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const { openfeature } = require('../dist/packages/openfeature-js/src');
+const { CloudbeesProvider } = require("../dist/packages/js-cloudbees-provider/src");
+
+/**
+ * Registers the environment variable provider to the globally scoped
+ * OpenFeature object.
+ */
+console.log('Registering the OpenFeature environment provider');
+openfeature.registerProvider(new CloudbeesProvider('618538bd7ebdeef023b5ff0d'));
```

Comment:
Could you please move this to an environment variable? Then update the existing `.env.example` file with an example.

Reply:
@beeme1mr done